### DISCUSSION TITLE
Can I use this as a wrapper for api?

### DISCUSSION THREAD

**Discussion by dionebigode** [cloud-py-api/nc_py_api#323] [Q&A]
I'm trying to access the Cospend API and after checking OSC API View I realized that the API is exposed!

![image](https://github.com/user-attachments/assets/d2238439-1228-4759-b09c-032a557714f0)

I thought of getting an Nextcloud api Wrapper to easily interface with it. Can I use cloud-py-api for this?

**Comment by bigcat88** [comment]
Yes, you can, there is an `ocs` method with which you can call these endpoints.

If you wish, you can even write higher-level functions using it and open a PR with them.
